src/client/pages/Thema/Erfpacht/ErfpachtDetail.module.scss

```diff
@@ -25,7 +25,7 @@
   white-space: nowrap;
 
   a {
-    vertical-align: middle;
+    vertical-align: 2px;
```

Comment:
Je kunt deze ook weghalen zie ik. Dan is de alignment goed.

Reply:
![Screenshot 2025-07-09 at 11 55 00](https://github.com/user-attachments/assets/97f60fe3-0eef-459d-a937-ad24dd18119d)